docs/1.3-supported-os.md

```diff
@@ -0,0 +1,40 @@
+# Composer 1.3 - Recommended OS versions
+
+The tables below provide recommended OS information for Composer 1.3.
+
+For this release, Composer will **only run on x64**. We will support more architectures in upcoming releases.
+
+## Windows
+
+OS                | Version            | Architectures |
+------------------|--------------------|---------------|
+Windows 10        | Version 1607+      | x64           |
+Windows Server    | 2012 R2+           | x64           |
+Nano Server       | Version 1803+      | x64           |
+Windows Client    | 7 (SP1), 8.1       | x64           |
+
+
+## macOS
+
+OS                | Version            | Architectures |
+------------------|--------------------|---------------|
+Mac OS X          | 10.14+             | x64           |
+
+
+## Linux
+
+OS                | Version            | Architectures |
+------------------|--------------------|---------------|
+Ubuntu            | 18.04, 20.04       | x64           |
+Debian            | 9+                 | x64           |
+
+
+## Supported Node.js versions
+
+If you are building Composer from source, the table below shows the Node.js versions that Composer will run on.
+
+Node.js |         Version            | Architectures |
+--------|----------------------------|---------------|
+LTS     | v10, v12, v14              | x64           |
+Non-LTS | v13                        | x64           |
```

Comment:
Should we avoid claiming composer will work in non-LTS node.

Reply:
I'll take it out for now.